test/library/packages/UnitTest/PopulateTests_PRIM/TestMultipleArgs.chpl

```diff
@@ -0,0 +1,20 @@
+use UnitTest;
+//verifies that functions with multiple argument aren't counted
+
+/* A normal function which takes a multiple argument*/
+proc dummy_test(test: Test, test2) throws {
+    //This is a dummy test
+}
+
+/* A normal function which takes a multiple argument*/
+proc dummy_test_default(test: Test, test2, test3 = 5) throws {
```

Comment:
Sorry, what I meant was more that only the `test: Test` argument wouldn't have a default value (meaning that it *could* be called with just the `test` argument, but wouldn't match the "takes only one argument" check that we have right now)

Reply:
Hey @lydia-duncan I am little confused in this. Really sorry for that.
Is this what you meant?
```chapel
proc dummy_test_default(test: Test, test2 = 5) throws { ... }
```

Reply:
Yup, that's exactly what I meant :)

Reply:
Okay, I have updated it.